### ISSUE TITLE
Add a switch to disable "top sites"

### ISSUE THREAD

**Issue by eatstorming** [brave/browser-ios#266]
My personal reasoning for this is just annoyance; the main page of an app I use for extended periods of time, changing based on what I've been browsing, annoys me.

However, there is also a privacy issue at play: an eavesdropper can see what you've been browsing if they manage to look at your screen when you open a new tab.

I understand that for a lot of people this may be a wanted feature. I just don't see why it can't be toggled on and off. So, please, add such an option to this otherwise awesome app.

Thank you.

**Comment by garvankeeley** [comment]
Good point, I'l see what I can do here

**Comment by bradleyrichter** [comment]
toggle in prefs would be great. We can just leave the default search icons until eventually allowing a bookmarks folder choice.

**Comment by garvankeeley** [comment]
My suggestion here is to have a setting for 
New tab shows:
[] Top sites
[] webpage: (UITextField to put address of page. Default text can be google.com)

**Comment by eatstorming** [comment]
Yes, please. That'd be awesome!

**Comment by bradleyrichter** [comment]
Good quick fix. SGTM!

**Comment by eatstorming** [comment]
Hello,

Has this been scrapped? :-(

**Comment by jhreis** [comment]
For reference, these are the current options from Brave's desktop browser.

<img width="325" alt="New tab page options on Brave Desktop browser" src="https://cloud.githubusercontent.com/assets/2781690/22473773/f0684b8e-e7a7-11e6-9692-faa68808aa86.png">

**Comment by jhreis** [comment]
@eatstorming, no, we have not scrapped it, although we currently do not have it scheduled for a release. Although not a complete substitute, you can remove specific websites from the new tab page by holding down on an icon, and clicking the "X":

![](https://cloud.githubusercontent.com/assets/2781690/22473997/a7ac68de-e7a8-11e6-8006-9b93fae5a752.png)

**Comment by DrSpaldo** [comment]
Hi, I also would like to see this one go through, since the desktop has it, it would be natural that the mobile version does as well.. thanks

**Comment by aecorredor** [comment]
Also, besides bookmarks, I would also like to add a favorite web site to the new tab page, so that the squared icon always appears there (where the facebook, yahoo, youtube, etc... icons are). This is a feature that safari has for example. I've been poking around to see if this feature exists already, but from what I have found it doesn't.

**Comment by DrSpaldo** [comment]
Hopefully this is added soon, it’s the main reason I can’t use Brave as my main browser :(

**Comment by brainlulz** [comment]
Any updates for the new Brave browser ?

**Comment by firesofmay** [comment]
Same here. I switched from Chrome and trying to use Speed Dial 2. But I can't get it to show as the default view for new tab. Any luck?

**Comment by jhreis** [comment]
@brainlulz We are hoping to release the beginning of December, but still a few unknowns. You can see how things are going over at https://github.com/brave/brave-ios/.

**Comment by jhreis** [comment]
@firesofmay Understood, we will move this the new repo (https://github.com/brave/brave-ios/issues/375) and will try to prioritize it after launch.

**Comment by saqib-ali** [comment]
Any update on this?
